deadlines/src/main/java/com/palantir/deadlines/Deadlines.java

```diff
@@ -177,6 +177,11 @@ private static void checkExpiration(long deadline, boolean internal) {
     // returns "1.523"
     @VisibleForTesting
     static String durationToHeaderValue(long durationNanos) {
+        if (durationNanos <= 0) {
```

Comment:
I think we could safely check `durationNanos < 1000000` because everything sub-1ms is encoded as `0.000`, but we can consider tweaking this separately

Reply:
ah good call, will fix